libsycl/docs/DesignDocs/UnifiedRuntime.rst

```diff
@@ -0,0 +1,26 @@
+=====================
+Unified Runtime
+=====================
+
+.. contents::
+   :local:
+
+.. _unified runtime:
+
+Overview
```

Comment:
We can avoid a sub-section (Overview) here. We can add this if we add more details to this document.

Thanks